### PR TITLE
Introduce users query

### PR DESCRIPTION
**Pull request by jalapl**
Screenshot 2020-07-09 at 16 54 25

### CODE REVIEW

File: app/graphql/types/query_type.rb [hunk 1]
```diff
@@ -2,10 +2,15 @@ module Types
   class QueryType < Types::BaseObject
     description "The query root of this schema"
 
+    field :users, [UserType], null: true
```
Comment (akaoss) [inline app/graphql/types/query_type.rb:5]:
```suggestion
    field :users, [UserType], null: false
```

File: app/graphql/types/query_type.rb [hunk 2]
```diff
@@ -2,10 +2,15 @@ module Types
...
     field :viewer, ViewerType, null: false
 
     def viewer
       {}
     end
+
+    def users
+      User.all
```
Comment (akaoss) [inline app/graphql/types/query_type.rb:13]: Let's sort them by name

File: app/graphql/types/query_type.rb (hunk 1 above)
Comment (jalapl) [reply app/graphql/types/query_type.rb:5]: 👍

File: app/graphql/types/query_type.rb (hunk 2 above)
Comment (jalapl) [reply app/graphql/types/query_type.rb:13]: 👍

**Pull request merged**